test/test.go

```diff
@@ -414,6 +417,47 @@ func (h *Helper) TempFile(path, contents string) {
 	h.Must(ioutil.WriteFile(filepath.Join(h.tempdir, path), bytes, 0644))
 }
 
+// GetTestFileBytes reads a file form the testdata directory into memory.  src is
+// relative to ./testdata.  Assumes tests take place starting in the cmd/dep
+// directory.
+func (h *Helper) GetTestFileBytes(src string) []byte {
+	content, err := ioutil.ReadFile(filepath.Join(h.origWd, "_testdata", src))
+	if err != nil {
+		panic(err)
+	}
+	if strings.HasSuffix(src, ".go") {
+		formatted, err := format.Source(content)
+		if err == nil {
+			content = formatted
+		}
+	}
+	return content
+}
+
+// GetTestFileString reads a file form the testdata directory into memory.  src is
+// relative to ./testdata.  Assumes tests take place starting in the cmd/dep
+// directory.
+func (h *Helper) GetTestFileString(src string) string {
+	return string(h.GetTestFileBytes(src))
+}
+
+// GetTestFileReader reads a file form the testdata directory into memory.  src is
+// relative to ./testdata.  Assumes tests take place starting in the cmd/dep
+// directory.
+func (h *Helper) GetTestFileReader(src string) io.Reader {
+	return strings.NewReader(h.GetTestFileString(src))
+}
+
+// TempCopy copies a temporary file from testdata into the temporary directory.
+// dest is relative to the temp directory location, and src is relative to
+// ./testdata.  Assumes tests take place starting in the cmd/dep directory.
+func (h *Helper) TempCopy(dest, src string) {
```

Comment:
It will be in the integration tests, where os proesses are called.  Still wrapping my head around the correct Go way to handle them.